Taskfile.yml

```diff
@@ -105,17 +105,46 @@ tasks:
       - mv {{.BINARY}} {{.LOCATION}}/{{.BINARY}}
 
   debug-start:
-    desc: Start headless debug
-    aliases: ["debug"]
+    desc: First, start headless debug
+    aliases: ["dbg-start", "dbg1", "debug1"]
     vars:
-      BINARY: terraform-provider-opslevel_{{.VERSION_DYNAMIC}}
+      BINARY: terraform-provider-opslevel-debug_{{.VERSION_DYNAMIC}}
+    cmds:
+      - defer: rm {{.BINARY}}
+      - go build -gcflags="all=-N -l" -ldflags="-X main.version={{.VERSION_DYNAMIC}}" -o {{.BINARY}} || exit 1
+      - chmod +x {{.BINARY}}
+      - task: terraform-init-from
+        vars: { INIT_DIR: "{{.WORKSPACE_DIR}}" }
+      - dlv exec --accept-multiclient --api-version=2 --continue --headless --wd={{.WORKSPACE_DIR}} {{.BINARY}} -- -debug | tee {{.DEBUG_LOG}}
+
+  debug-attach:
```

Comment:
Should we add this to this task?  https://taskfile.dev/usage/#interactive-cli-application

Reply:
Ohhh yeah I'll test this out 🚀

Reply:
I think the warning prompt would only add a minor annoyance

Reply:
Added `interactive: true` 👍